### PR TITLE
Logs: add stream input vars

### PR DESCRIPTION
**Pull request by mtojek**
This PR adds variables to logs stream input.

~~Blocker - I'm opening this as draft, because there is a design issue to be solved. If both, metricbeat and filebeat, have a dataset with the same name (e.g. aws - elb), the `dataset/manifest.yml` can't define two different types.~~

see `cloudtrail/manifest.yml`:

```yaml
title: Aws Cloudtrail Logs
release: beta
type: logs
streams:
- input: logs
  vars:
  - default: s3
    name: input
  - name: queue_url
  - name: shared_credential_file
  - name: credential_profile_name
  - name: visibility_timeout
  - name: api_timeout
  - name: endpoint
  title: Aws cloudtrail logs
  description: Collect Aws cloudtrail logs
``` 
~~As there is no metricbeat module for `cloudtrail`, this file hasn't been overridden (opposite to some other aws datasets).~~

Fixed by creating two directories for datasets having the same name (e.g. `cloudwatch-metrics` and `cloudwatch-logs`).

BTW. Package validation passed:

```bash
$ go run ./dev/generator/ -sourceDir=./dev/packages/beats/ -publicDir=./public -tarGz=true
```

BTW2. Packages are properly pulled and presented by Kibana's ingestManager

```bash
$ yarn es snapshot -E xpack.security.authc.api_key.enabled=true
```

```bash
$ yarn start --xpack.ingestManager.enabled=true --xpack.ingestManager.epm.enabled=true --xpack.ingestManager.fleet.enabled=true --xpack.ingestManager.epm.registryUrl=http://localhost:8080/
```

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var (
 
 	publicDir    = "./public"
 	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/generated/", "./dev/packages/example/"}
+	packagePaths = []string{"./dev/packages/generated/", "./dev/packages/example/", "./dev/packages/beats/"}
```
Comment (ruflin) [inline magefile.go]: I'm thinking if we should add it here already or not. I like that it does validate the packages. But I'm wondering if they should already served. The good news is I assume things like nginx and system package keep working, as they have a never version number.

Comment (ruflin) [reply magefile.go]: @mtojek To get this PR in, can you remove this and merge it without it?

Comment (mtojek) [reply magefile.go]: Sure, not a problem. Reverted.

File: dev/packages/beats/activemq-0.0.1/dataset/audit/manifest.yml [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
 release: ga
 type: logs
+streams:
+- input: logs
+  vars:
+  - default:
+    - /opt/apache-activemq-*/data/audit.log*
+    name: paths
+    os.darwin:
+    - /usr/local/apache-activemq-*/data/audit.log*
```
[the remark this answers is not in the document]
Comment (mtojek) [reply dev/packages/beats/activemq-0.0.1/dataset/audit/manifest.yml]: Sorry about that. Fixed.

**Pull request merged**